### PR TITLE
Save selected text in store

### PR DESCRIPTION
**Pull request by davidmoore1**
A follow up PR will need to be added to prevent the verse number and footnote text from being displayed.  Comment code is in the verseSelectUtil.ts that shows how you can parse through the selected verses in order

### CODE REVIEW

File: src/lib/data/stores.js [hunk 1]
```diff
@@ -131,67 +131,69 @@ function createSelectedVerses()  {
...
-            references.push(reference);
-            external.set(references);
+            let selections = get(external);
+            selections.push(selection);
+            external.set(selections);
         },
         removeVerse: (id) => {
             const currentRefs = get(refs);
-            const reference = {
+            const selection = {
```
[the remark this answers is not in the document]
Comment (chrisvire) [reply src/lib/data/stores.js:149]: SO ... what does 'Done' mean? Did you change something?

File: src/lib/data/stores.js [hunk 2]
```diff
@@ -131,67 +131,69 @@ function createSelectedVerses()  {
     
     return {
         subscribe: external.subscribe,
-        addVerse: (id) => {
+        addVerse: (id, text) => {
```
[the remark this answers is not in the document]
Comment (davidmoore1) [reply src/lib/data/stores.js:134]: Done

File: src/lib/data/stores.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidmoore1) [reply src/lib/data/stores.js:149]: Removed selection and the currentRefs line.  They were basically from cut and paste of the add

**Review by chrisvire — APPROVED** [review]
LGTM

**Pull request merged**